lib/iris/tests/unit/util/test_mask_cube.py

```diff
@@ -29,24 +26,27 @@ def full2d_global():
 
 
 class MaskCubeMixin:
-    def assertOriginalMetadata(self, cube, func):
+    def assertOriginalMetadata(self, request, cube, func):
         """Check metadata matches that of input cube.  func is a string indicating
         which function created the original cube.
 
         """
         reference_dir = pathlib.Path("unit/util/mask_cube")
         reference_fname = reference_dir / f"original_cube_{func}.cml"
-        self.assertCML(
+        _shared_utils.assert_CML(
+            request,
```

Comment:
```suggestion
            self.request,
```

Since this is a method it should always be able to access `self.request` (or error if it has not been set).

That would also simplify all the calls to the method since you don't need `request` as an argument.

Reply:
The failing tests are just because you're still passing `self.request` in various calls and you don't need to anymore.